tests/test_nonbonded.py

```diff
@@ -417,59 +415,44 @@ def test_nonbonded(self):
         np.random.seed(4321)
         D = 3
 
-        benchmark = False
+        for size in [33, 231, 1050]:
 
-        # test numerical accuracy on a box of water
+            _, coords, box, _ = builders.build_water_system(6.2)
+            coords = coords/coords.unit
+            coords = coords[:size]
 
-        for size in [33, 231, 1050]:
+            N = coords.shape[0]
 
-            if not benchmark:
```

Comment:
There's other benchmarking tests now in test_nonbonded so this jank is no longer needed.